Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ccl/holder/grouping/DccGroupingExtensionsTest.kt

```diff
@@ -0,0 +1,299 @@
+package de.rki.coronawarnapp.ccl.holder.grouping
+
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CertificatePersonIdentifier
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CwaCovidCertificate
+import de.rki.coronawarnapp.util.dcc.groupByPerson
+import io.kotest.matchers.collections.shouldContainAll
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Test
+import testhelpers.BaseTestInstrumentation
+
+class DccGroupingExtensionsTest : BaseTestInstrumentation() {
+
+    @Test
+    fun `first grouping check`() {
+        val certificatesList = setOf(
+            certA1,
+            certA2,
+            certA3,
+            certA4,
+            certA5,
+            certA6,
+            certA7,
+            certA9,
+            certA8,
+            certB1,
+            certB2,
+            certB3,
+            certC1
+        )
+
+        val result = certificatesList.toSet().groupByPerson()
```

Comment:
`.toSet()` is not necessary I think? Can be removed (also in other places in this file)